scripts/node_e2e_installer

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+NODE_E2E_COMMIT=4d14fd61d2d533a0bb81557b54d3505be830efa7  # Commit to run upstream node e2e tests
+
+enable_selinux () {
+    # Make sure SELinux is enabled
+    setenforce 1
```

Comment:
This certainly must be @rhatdan approved coding techniques!  ;^)

Reply:
I recently wrote my first SELinux module, AMA :grin: